apps/prairielearn/src/pages/instructorIssues/instructorIssues.ejs

```diff
@@ -14,22 +14,23 @@
     <main id="content" class="container-fluid">
       <%- include('../partials/courseSyncErrorsAndWarnings'); %>
       <% if (authz_data.has_course_permission_edit) { %>
-      <div class="modal fade" id="closeAllIssuesModal" tabindex="-1" role="dialog" aria-labelledby="closeAllIssuesModalLabel">
+      <div class="modal fade" id="closeMatchingIssuesModal" tabindex="-1" role="dialog" aria-labelledby="closeMatchingIssuesModalLabel">
         <div class="modal-dialog" role="document">
           <div class="modal-content">
             <div class="modal-header">
-              <h4 class="modal-title" id="closeAllIssuesModalLabel">Close all issues</h4>
+              <h4 class="modal-title" id="closeMatchingIssuesModalLabel">Close matching issues</h4>
             </div>
             <div class="modal-body">
-              Are you sure you want to close all issues? This cannot
-              be undone.
+              Are you sure you want to close <strong><%= rows.length %></strong> matching
+              <%= rows.length === 1 ? 'issue' : 'issues' %>?
             </div>
             <div class="modal-footer">
-              <form name="close-all-form" method="POST">
-                <input type="hidden" name="__action" value="close_all">
+              <form name="close-matching-form" method="POST">
+                <input type="hidden" name="__action" value="close_matching">
                 <input type="hidden" name="__csrf_token" value="<%= __csrf_token %>">
+                <input type="hidden" name="unsafe_issue_ids" value="<%= rows.map(row => row.issue_id).join(',') %>">
```

Comment:
Instructor perspective: I would prefer if the action closed ALL matching issues so I don't have to page through one-by-one. Usually this happens when it's a broken question and it's not worth going through all of them. 

FWIW I think Google does this best when handling email:

![image](https://github.com/PrairieLearn/PrairieLearn/assets/2507459/c9570084-9a29-4a08-9060-2255b401b4fe)

That said, this feature is already much needed as-is so IMHO is worth proceeding without the frills.

Reply:
I'd say make a quick note about it in the modal body (i.e., if `issueCount > rows.length` then add a note to the effect of "Only the <%= rows.length %> issues in the current page will be closed", or something similar. Or just keep it as is, I don't have a strong opinion.

Reply:
I added such a note in a973fb8 and refined it in d751cec.

I agree that something like Gmail's approach would be great here; I think GitHub's notifications page does this well too. Adding checkboxes and the necessary client-side JS to drive all that is a bit outside the scope of this PR, but I'd like to get to that eventually. Firas, like you said, this is already a nice improvement, even if it isn't yet as good as it could be!